catalog/app/constants/config.ts

```diff
@@ -0,0 +1,3 @@
+import { getGlobalConfig } from 'utils/Config'
+
+export default getGlobalConfig()
```

Comment:
well, i wasn't thinking in such terms, i was thinking a constant is just a value which is shared globally across the system and can be imported wherever you need it. like, i can put the config into some module in `utils/`, but i think it makes quite a lot of sense to have it in `constants/`

Reply:
new top-level namespace to do `import cfg from 'config'`? well, it kinda makes sense, but it's more work because we need to adjust webpack and typescript setup